tests/Aql/ExecutorTestHelper.h

```diff
@@ -244,19 +312,52 @@ struct ExecutorTestHelper {
     return *this;
   }
 
-  auto run(typename E::Infos infos) -> void {
+  auto run() -> void {
     ResourceMonitor monitor;
     AqlItemBlockManager itemBlockManager(&monitor, SerializationFormat::SHADOWROWS);
 
     auto inputBlock = generateInputRanges(itemBlockManager);
 
-    auto testeeNode = std::make_unique<SingletonNode>(_query.plan(), 1);
+      _testee->addDependency(inputBlock.get());
```

Comment:
Can you unify run and runPipeline?

e.g.:
run() -> 
ASSERT pipeline is empty.
Insert 1 node in _pipeline.
call runPipeline()

I think they should be identical then